### PR TITLE
Refs CRM-21234 Missing subdivisions of Tajikistan.

### PR DESCRIPTION
**Pull request by nowszy**
Overview
----------------------------------------
 * [CRM-21234: Missing subdivisions of Tajikistan](https://issues.civicrm.org/jira/browse/CRM-21234)

### CODE REVIEW

File: CRM/Upgrade/Incremental/sql/4.7.25.mysql.tpl [hunk 1]
```diff
@@ -20,3 +20,8 @@ FROM civicrm_uf_join
 LEFT JOIN civicrm_event e on civicrm_uf_join.entity_id = e.id
 WHERE (civicrm_uf_join.module = 'CiviEvent' OR civicrm_uf_join.module = 'CiviEvent_Additional')
 AND e.id IS NULL;
+
+-- CRM-21234 Missing subdivisions of Tajikistan.
+ INSERT INTO civicrm_state_province (id, country_id, abbreviation, name) VALUES
+   (NULL, 1209, "DU", "Dushanbe"),
+   (NULL, 1209, "RA", "Nohiyahoi Tobei Jumhur�");
```
Comment (monishdeb) [inline CRM/Upgrade/Incremental/sql/4.7.25.mysql.tpl]: @nowszy the master is currently pointing at 4.7.26, so can you move this upgrade code to ```CRM/Upgrade/Incremental/sql/4.7.26.mysql.tpl``` !!

**Comment by seamuslee001** [comment]
Jenkins ok to test

**Comment by seamuslee001** [comment]
@eileenmcnaughton @colemanw this looks good to merge to me. I haven't tested the upgrade but looks sound to me.

**Comment by monishdeb** [comment]
Merged  :)

**Pull request merged**